guides/common/attributes-base.adoc

```diff
@@ -15,7 +15,7 @@
 :ProvisioningDocURL: {BaseURL}Provisioning_Hosts/{BaseFilenameURL}#
 :ReleaseNotesURL: {BaseURL}Release_Notes/{BaseFilenameURL}#
 :TuningDocURL: {BaseURL}Tuning_Performance/{BaseFilenameURL}#
-:UpgradingDocURL: {BaseURL}Upgrading_and_Updating/{BaseFilenameURL}#
+:UpgradingDocURL: {BaseURL}Upgrading_Project/{BaseFilenameURL}#
```

Comment:
FYI, the DocURLs may require changes in code repos to update links to the documentation, which would have to be done by developers. This goes for both upstream and downstream. I don't expect it for these guides, but you should verify it with the devs in your team to make sure. It isn't a blocker for this PR, though.

Reply:
Thanks for this @Lennonka 
I will sync with respective teams and take action if required. :+1: